packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -99,6 +101,7 @@ export class SearchBoxBase extends BaseComponent<ISearchBoxProps, ISearchBoxStat
               iconProps={{ iconName: 'Clear' }}
               {...clearButtonProps}
               onClick={this._onClearClick}
+              ariaLabel={clearButtonAriaLabel}
```

Comment:
Could this be passed in through clearButtonProps?

Reply:
Definitely! Is there a common practice for specifying default values in the .types.tsx file for something like IButtonProps?

In other words, for `clearButtonAriaLabel?: string` I have the comment `@default 'Clear text'`. 

Should I write something in the comment above `clearButtonProps?: IButtonProps`

Reply:
Done! I followed the pattern used in Dialog for `modalProps`. Let me know what you think.